.storybook/webpack.config.js

```diff
@@ -75,5 +75,21 @@ module.exports = async ( { config } ) => {
 
 	config.module.rules.push( mainConfig.svgRule );
 
+	config.module.rules.push( {
```

Comment:
Note, the published `@material/web@0.1.0-alpha.0` package uses the `??` nullish coalescing operator, necessitating some additional rules to our webpack configs.